src/player.py

```diff
@@ -2,9 +2,61 @@
 # currently.
 
 class Player:
-    def __init__(self,name, current_room):
+    def __init__(self,name, startingRoom):
         self.name = name
-        self.current_room = current_room
+        self.currentRoom = startingRoom
+        self.items = []
+        self.capability = ['take','drop']
+
 
     def __str__(self):
-        return(f'{self.name} in {self.current_room} room.')
+        return(f'\n{self.name} with possesson:{self.items} in\n{self.currentRoom}')
+
+    def travel(self, direction):
+        nextRoom = self.currentRoom.goNextRoom(direction)
+        if nextRoom is not None:
+            self.currentRoom = nextRoom
+            print(self.currentRoom)
+        else:
+            print("You cannot move in that direction.")
+
+    def action(self, cmd):
+        cmdItemList = []
+        actionCmd = None
+        tokenList = cmd.split()
+
+        for token in tokenList:
+            if token in self.capability :
+                actionCmd = token
+            else:
+                cmdItemList.append(token.lower())
+
+        if actionCmd == 'take':
+            for takeItem in cmdItemList:
+                itemTaken=False
+                #self.currentRoom.items returns item objects list.
+                # roomItem extracts item name from each item objects
+                for roomItem in self.currentRoom.items:
+                    if roomItem.name == takeItem:
+                        self.items.append(roomItem)
+                        itemTaken=roomItem.on_take()
+                        break
```

Comment:
good call on the break after item has been found. Look at ways to manage exceptions and errors